iothub/device/src/Utilities/ObjectConversionHelper.cs

```diff
@@ -20,8 +20,7 @@ internal static bool TryCastOrConvert<T>(object objectToCastOrConvert, PayloadCo
             {
                 // If the cannot be cast to <T> directly we need to try to convert it using the serializer.
                 // If it can be successfully converted, go ahead and return it.
```

Comment:
I believe this comment needs an update as well, any advice? @drwill-ms

Reply:
Yes